functions/census-import/census-import.data.service.js

```diff
@@ -0,0 +1,59 @@
+const moment = require('moment')
+const mssql = require('mssql')
+const uuidv4 = require('uuid/v4')
+
+const config = require('../config')
+
+const schema = '[mtc_admin]'
```

Comment:
unnecessary IMO.

ive seen this in sql service and its not proven worthwhile.  we arent going to change the schema name.

Reply:
@GuyHarwood removed const since its included in the name now on a1a933d